challenger-deep-theme.el

```diff
@@ -383,12 +383,12 @@
 				    (,ct (:background, ct-blue-d :foreground, ct-black))))
 
      ;; diff-hl
-     `(diff-hl-change              ((,c (:foreground ,vc-modified))
-                                    (,ct (:foreground ,ct-vc-modified))))
-     `(diff-hl-delete              ((,c (:foreground ,vc-deleted))
-                                    (,ct (:foreground ,ct-vc-deleted))))
-     `(diff-hl-insert              ((,c (:foreground ,vc-added))
-                                    (,ct (:foreground ,ct-vc-added))))
+     `(diff-hl-change              ((,c (:foreground ,vc-modified :background ,vc-modified))
```

Comment:
I think we should make the foreground and the background in different colors here in order to support displaying a plus/minus sign. (See Screenshot: https://github.com/dgutov/diff-hl#diff-hl-margin-mode).
We could either use the dark variants (e.g. add `ct-red-d` as `vc-deleted-d`), or use white/black as the foreground color.

Reply:
Oh, I had not considered it. Thanks for pointing out.

After testing, I find using `yellow` as foreground for `diff-hl-change` works fine. (But that's weird that `vc-modified` is using the dark variant `yellow-d`, but `vc-insert` and `vc-delete` are using normal variant.)

Or we can keep consistent with terminal color since it's more visible.

I think the latter is better. And I find `:foreground` doesn't work in terminal, so is it necessary to keep it in code?

I'll put the comparison below.